src/Security/Authentication/OpenIdConnect/src/OpenIdConnectConfigureOptions.cs

```diff
@@ -0,0 +1,136 @@
+        options.BackchannelTimeout = StringHelpers.ParseValueOrDefault(options.BackchannelTimeout, configSection[nameof(options.BackchannelTimeout)], _invariantTimeSpanParse);
+        options.CallbackPath = new PathString(configSection[nameof(options.CallbackPath)] ?? options.CallbackPath);
+        options.ClaimsIssuer = configSection[nameof(options.ClaimsIssuer)];
+        options.ClientId = configSection[nameof(options.ClientId)];
+        options.ClientSecret = configSection[nameof(options.ClientSecret)];
+        options.CorrelationCookie = GetCookieFromConfig(configSection.GetSection(nameof(options.CorrelationCookie)), options.CorrelationCookie);
+        options.DisableTelemetry = StringHelpers.ParseValueOrDefault(options.DisableTelemetry, configSection[nameof(options.DisableTelemetry)], bool.Parse);
+        options.ForwardAuthenticate = configSection[nameof(options.ForwardAuthenticate)];
+        options.ForwardChallenge = configSection[nameof(options.ForwardChallenge)];
+        options.ForwardDefault = configSection[nameof(options.ForwardDefault)];
+        options.ForwardForbid = configSection[nameof(options.ForwardForbid)];
+        options.ForwardSignIn = configSection[nameof(options.ForwardSignIn)];
+        options.ForwardSignOut = configSection[nameof(options.ForwardSignOut)];
+        options.GetClaimsFromUserInfoEndpoint = StringHelpers.ParseValueOrDefault(options.GetClaimsFromUserInfoEndpoint, configSection[nameof(options.GetClaimsFromUserInfoEndpoint)], bool.Parse);
+        options.MapInboundClaims = StringHelpers.ParseValueOrDefault(options.MapInboundClaims, configSection[nameof(options.MapInboundClaims)], bool.Parse);
+        options.MaxAge = options.MaxAge is TimeSpan maxAge ? StringHelpers.ParseValueOrDefault(maxAge, configSection[nameof(options.MaxAge)], _invariantTimeSpanParse) : options.MaxAge;
+        options.MetadataAddress = configSection[nameof(options.MetadataAddress)];
+        options.NonceCookie = GetCookieFromConfig(configSection.GetSection(nameof(options.NonceCookie)), options.NonceCookie);
+        options.Prompt = configSection[nameof(options.Prompt)];
+        options.RefreshInterval = StringHelpers.ParseValueOrDefault(options.RefreshInterval, configSection[nameof(options.RefreshInterval)], _invariantTimeSpanParse);
+        options.RefreshOnIssuerKeyNotFound = StringHelpers.ParseValueOrDefault(options.RefreshOnIssuerKeyNotFound, configSection[nameof(options.RefreshOnIssuerKeyNotFound)], bool.Parse);
+        options.RemoteAuthenticationTimeout = StringHelpers.ParseValueOrDefault(options.RemoteAuthenticationTimeout, configSection[nameof(options.RemoteAuthenticationTimeout)], _invariantTimeSpanParse);
+        options.RemoteSignOutPath = new PathString(configSection[nameof(options.RemoteSignOutPath)] ?? options.RemoteSignOutPath);
```

Comment:
Avoid the implicit string converter:
```suggestion
        options.RemoteSignOutPath = new PathString(configSection[nameof(options.RemoteSignOutPath)] ?? options.RemoteSignOutPath?.Value);
```